handlers/command_handler.py

```diff
@@ -54,8 +55,9 @@ def _delete_first_two_words(self, phrase: str) -> str:
 You're also welcome to disable the feature, however, we recommend you keep it enabled to keep the evil cogs out!
 To find your IP address, click here to use Google's IP checker: https://goo.gl/search/ip
 If you're having trouble locating the "Trusted IP's" section of the website, please refer to the following image: https://imgur.com/a/35LuQ
-"""
-        await self.client.send_message(message.channel, ip_help)
+""".format(message)
+
+        await self.client.send_message(discord.Object(id='347411900864135189'), ip_help)
```

Comment:
What is ID 347411900864135189? If you're trying to send a DM instead of replying in the channel, you can 
```python
await self.client.send_message(message.author, str)
```

Reply:
ID 347411900864135189 is the ToonHQ channel on the discord. I've changed this to ask for a variable instead, which can be found at the top of the script.